### PR TITLE
add docs for stepper

### PR DESCRIPTION
**Pull request by Spartan-71**
Issue reference : [Link](https://github.com/reflex-dev/reflex/issues/3182)

### **Added two examples:**

**1] Basic** 

image

**2] Used on_click event handler :**

image

### CODE REVIEW

**Comment by elviskahoro** [comment]
@Spartan-71 can you rebase this on main! Just merged a fix for the bug that was blocking this PR!

**Comment by Spartan-71** [comment]
@elviskahoro I am done with rebasing , if any things wrong plz let me know.

**Comment by elviskahoro** [comment]
@Spartan-71 can you rebase one more time! We cut a new release for reflex-chakra: bumped to 0.6.2

**Comment by Spartan-71** [comment]
Done.

**Comment by Spartan-71** [comment]
@elviskahoro @masenf plz have a look on my PR.

File: docs/rcweb/rcweb/markdowns/navigation/stepper.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    index=1,
+    orientation="horizontal",
+    width="700px",
+    gap='10px',
+)
+```
+
+Below is an example on how `on_click` event handler can be used to set an `step_status` to `active`.
+
+```python demo
```
Comment (masenf) [inline docs/rcweb/rcweb/markdowns/navigation/stepper.md:66]:
```suggestion
```python demo exec
```

when you're defining state, i think you need to use `python demo exec` block to ensure the state gets registered

File: docs/rcweb/rcweb/markdowns/navigation/stepper.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+class ProgressBar(rx.State):
+    index: int =1
+
+    def setactive(self,value :int):
+        self.index=value
```
Comment (masenf) [inline docs/rcweb/rcweb/markdowns/navigation/stepper.md:71]:
```suggestion
    index: int = 1

    def setactive(self, value: int):
        self.index = value
```

doc examples should follow standard formatting practices (ruff/black)

**Review by ElijahAhianyo — APPROVED** [review]

File: docs/rcweb/rcweb/markdowns/navigation/stepper.md (hunk 2 above)
Comment (Spartan-71) [reply docs/rcweb/rcweb/markdowns/navigation/stepper.md]: I have formatted all the examples using black and made a commit.

**Review by masenf — APPROVED** [review]

**Comment by Spartan-71** [comment]
Hey @picklelo @elviskahoro, since this repo isn’t participating in Hacktoberfest, my PR isn’t counting towards it. Could you please add the `hacktoberfest-accepted` label to this PR?

**Comment by elviskahoro** [comment]
Added the label! Thanks again @Spartan-71 
join our discord too!

**Pull request merged**